src/Console/Commands/EventStoreWorker.php

```diff
@@ -150,7 +150,7 @@ private function makeSerializableEvent(EventRecord $event): JsonEventRecord
         $data->setEventNumber($event->getNumber());
         $data->setData(json_encode($event->getData()));
         $data->setEventStreamId($event->getStreamId());
-        $data->setMetadata(json_encode($this->safeGetMetadata($event)));
+        $data->setMetadata(json_encode($this->safeGetMetadata($event) ?? []));
```

Comment:
Would this be better in the safeGetMatadata function?

Reply:
Yup - moved

Reply:
Yup - moved